tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -207,6 +207,76 @@ error: Cannot access private member \\"private_static_method\\" of \\"Foo\\"
 
  
  
+Tests 1 failed (1)
+Test Files 1 failed (1)
+Duration <DURATION>"
+`;
+
+exports[`access_private_apis.test.w 1`] = `
+"error: Class \\"subdir2.MyPrivateClass\\" is private
+  --> ../../../examples/tests/invalid/access_private_apis.test.w:3:5
+  |
+3 | new subdir2.MyPrivateClass();
+  |     ^^^^^^^^^^^^^^^^^^^^^^ Class \\"subdir2.MyPrivateClass\\" is private
+  |
+  --> ../../../examples/tests/invalid/subdir2/file.w:1:7
+  |
+1 | class MyPrivateClass {}
+  |       -------------- defined here
+
+
+error: Class \\"subdir2.inner.MyOtherPrivateClass\\" is private
+  --> ../../../examples/tests/invalid/access_private_apis.test.w:6:5
+  |
+6 | new subdir2.inner.MyOtherPrivateClass();
+  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Class \\"subdir2.inner.MyOtherPrivateClass\\" is private
+  |
+  --> ../../../examples/tests/invalid/subdir2/inner/blah.w:1:7
+  |
+1 | class MyOtherPrivateClass {}
+  |       ------------------- defined here
+
+
+error: Interface \\"subdir2.MyPrivateInterface\\" is private
+   --> ../../../examples/tests/invalid/access_private_apis.test.w:11:14
+   |
+11 | class A impl subdir2.MyPrivateInterface {}
+   |              ^^^^^^^^^^^^^^^^^^^^^^^^^^ Interface \\"subdir2.MyPrivateInterface\\" is private
+   |
+   --> ../../../examples/tests/invalid/subdir2/file.w:7:11
+   |
+ 7 | interface MyPrivateInterface {}
+   |           ------------------ defined here
+
+
+error: Expected an interface, instead found type \\"unresolved\\"
+   --> ../../../examples/tests/invalid/access_private_apis.test.w:11:14
+   |
+11 | class A impl subdir2.MyPrivateInterface {}
+   |              ^^^^^^^^^^^^^^^^^^^^^^^^^^ Expected an interface, instead found type \\"unresolved\\"
+
+
+error: Struct \\"subdir2.MyPrivateStruct\\" is private
+   --> ../../../examples/tests/invalid/access_private_apis.test.w:14:9
+   |
+14 | let s = subdir2.MyPrivateStruct {};
+   |         ^^^^^^^^^^^^^^^^^^^^^^^ Struct \\"subdir2.MyPrivateStruct\\" is private
+   |
+   --> ../../../examples/tests/invalid/subdir2/file.w:3:8
+   |
+ 3 | struct MyPrivateStruct {}
+   |        --------------- defined here
+
+
+error: Expected identifier \\"subdir2\\" to be a variable, but it's a namespace
+   --> ../../../examples/tests/invalid/access_private_apis.test.w:17:9
+   |
+17 | let e = subdir2.MyPrivateEnum.A;
+   |         ^^^^^^^ Expected identifier \\"subdir2\\" to be a variable, but it's a namespace
```

Comment:
This isn't the ideal error message (we really want to say that "MyPrivateEnum" was found, but it's private), but I think the correct error message gets thrown away when we're converting `subdir2.MyPrivateEnum.A` from a Reference::InstanceMember to a Reference::TypeMember

Reply:
This error pops up in other places where it's not really helpful (https://github.com/winglang/wing/issues/2876) so I think it's fine for this PR as-is